packages/components/src/Toggle/Toggle.component.js

```diff
@@ -33,6 +34,7 @@ function Toggle({ id, onChange, onBlur, label, checked, autoFocus, disabled, cla
 					checked={checked}
 					autoFocus={autoFocus}
 					disabled={disabled}
+					{...omit(rest, 'data-feature')}
```

Comment:
in that case please remove useless destructured from props:
* onChange
* onBlur,
* autoFocus

they will be in the rest  which I prefer call props
> {id, checked, className, disabled, ...props}